data/avro-schema/src/main/resources/avro/net/corda/data/p2p/markers/AppMessageMarker.avsc

```diff
@@ -1,19 +1,19 @@
 {
   "type": "record",
   "name": "AppMessageMarker",
-  "namespace": "net.corda.p2p.markers",
+  "namespace": "net.corda.data.p2p.markers",
   "fields": [
     {
       "name": "marker",
       "type": [
-        "net.corda.p2p.markers.LinkManagerReceivedMarker",
-        "net.corda.p2p.markers.LinkManagerSentMarker",
-        "net.corda.p2p.markers.ApplicationProcessedMarker",
-        "net.corda.p2p.markers.ApplicationRejectedStaleMarker",
-        "net.corda.p2p.markers.GatewaySentMarker",
-        "net.corda.p2p.markers.LinkManagerProcessedMarker",
-        "net.corda.p2p.markers.LinkManagerDiscardedMarker",
-        "net.corda.p2p.markers.TtlExpiredMarker"
+        "net.corda.data.p2p.markers.LinkManagerReceivedMarker",
+        "net.corda.data.p2p.markers.LinkManagerSentMarker",
+        "net.corda.data.p2p.markers.ApplicationProcessedMarker",
+        "net.corda.data.p2p.markers.ApplicationRejectedStaleMarker",
+        "net.corda.data.p2p.markers.GatewaySentMarker",
+        "net.corda.data.p2p.markers.LinkManagerProcessedMarker",
+        "net.corda.data.p2p.markers.LinkManagerDiscardedMarker",
+        "net.corda.data.p2p.markers.TtlExpiredMarker"
```

Comment:
Looks line in the past we forgot to update the doc on line 25 of this file should be `net.corda.p2p.app.AuthenticatedMessage` (I can't comment on line 25).

Reply:
what a catch! I'll fix it